src/AutoRest.CSharp/AutoRest/Plugins/CSharpProj.cs

```diff
@@ -29,6 +28,21 @@ internal class CSharpProj : IPlugin
     <PackageReference Include=""Azure.Core"" Version=""1.11.0"" />
   </ItemGroup>
 
+</Project>
+";
+        private string _armCsProjContent = @"<Project Sdk=""Microsoft.NET.Sdk"">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+    <Nullable>annotations</Nullable>
+  </PropertyGroup>
+{0}
```

Comment:
I don't think you ever pass a value to this parameter.

Reply:
yeah we just copied what _csProjContent did, both pass in "" if you look at line 109

Reply:
Right, but it's used in _csProjContent